### DISCUSSION TITLE
PCare

### DISCUSSION THREAD

**Discussion by kibokib** [virusphp/service-bridging#10] [General]
permisi, saya lagi belajar utk bridging BPJS Pcare.. apakah bisa memakai Lib ini? mohon pencerahan

Terima Kasih sebelumnya

**Comment by virusphp** [comment]
terimaksih sudah menghubungi, untuk briding pcare sepertinya belum support kak karena keterbasan cons untuk pcare sehingga saya belum sempat membuat untuk versi pcare
